_sources/TransformingSequences/week4a1.rst

```diff
@@ -18,10 +18,10 @@ Chapter Assessment - List Methods
    :answer_a: I.
    :answer_b: II.
    :answer_c: Neither is the correct reference diagram.
-   :feedback_a: Yes, when we are using the remove method, we are just editing the existing list, not making a new copy.
-   :feedback_b: When we use the remove method, we just edit the existing list. We do not make a new copy that does not include the removed object.
+   :feedback_a: When we use the remove method, we just edit the existing list. We do not make a new copy that does not include the removed object.
+   :feedback_b: Yes, when we are using the remove method, we are just editing the existing list, not making a new copy. 
    :feedback_c: One of the diagrams is correct - look again at what is happening to lst.
-   :correct: a
```

Comment:
a is the correct answer.  the image for b is trying to show that the list has been copied.